.travis.yml

```diff
@@ -17,6 +17,7 @@ before_script:
 - npm install -g jshint
 script:
 - bundle exec rake spec
+- bundle exec rubocop
```

Comment:
I wonder if we should use CodeClimate to run the cops; IMHO it gives better feedback on Github PRs: you get two CI results. Travis may run the specs just fine, it may "just" be coding style that needs improvement.

I have never set up CodeClimate with custom rules, though. If we cannot reuse our rubocop.yml, it's not worth pursuing this.

Reply:
I think it works like this: we add a codeclimate.yml where we enable rubocop  (and a js linter), and then cc uses our rubocop.yml for the rules.

Reply:
If it is that easy, I'd prefer CC over running it on Travis

Reply:
Just for the sake of completeness: I just came across [build stages](https://docs.travis-ci.com/user/build-stages) for Travis. Would also solve the problem I think.

Reply:
I'll make a separate issue to discuss/decide on the CodeClimate way. Get it out of the way of merging this. 🎉